qml/pages/ProgramDelegate.qml

```diff
@@ -55,6 +67,7 @@ ListItem {
             font.bold: true
             font.pixelSize: Theme.fontSizeExtraSmall
             truncationMode: TruncationMode.Fade
+            y: parent.y + Theme.paddingSmall
```

Comment:
Maybe use topMargin in anchors below instead? Seemed work the same way and is a bit nicer way.

Reply:
That seems nicer option indeed. I've had some issues with anchors before, so I went manually with the y value.